web/assets/styles/common.scss

```diff
@@ -1,6 +1,7 @@
 html,
 body {
-    background: var(--body-bg);
+    background: #000;
+    background: var(--body-bg, #000);
```

Comment:
Hey Yannick,
I suspect it's not because of SASS but [cssmin package](https://www.npmjs.com/package/gulp-cssmin). In the docs it references [clean-css](https://github.com/jakubpawlowicz/clean-css).

We can try to change [this line](https://github.com/kubawolanin/org.openhab.ui.habpanel/blob/366a61df2f779e8e66d0595fa28d31141f654a63/web/gulpfile.js#L50) like so:

```js
        .pipe(sass())
        .pipe(cssmin({
            removeDuplicateRules: false
        }))
        .pipe(rename({
```

And see if that could help.
I don't have my dev environment with me atm, so can't try it myself now.